link-verifier/verify-links.py

```diff
@@ -335,7 +335,9 @@ def main():
             dirs[:] = [dir for dir in dirs if dir.lower() not in exclude_dirs]
             for file in files:
                 if any(file.endswith(file_type) for file_type in args.include_files):
-                    with open(os.path.join(root, file), 'r') as f:
+                    f_path = os.path.join(root, file)
+                    print("Processing File: {}".format(f_path))
+                    with open(f_path, 'r', encoding="utf8", errors='ignore') as f:
```

Comment:
A comment here would be useful about the support for special characters with UTF-8 configuration for the file read